src/g_inventory/a_weapons.cpp

```diff
@@ -324,7 +324,7 @@ bool AWeapon::DepleteAmmo(bool altFire, bool checkEnough, int ammouse)
 {
 	IFVIRTUAL(AWeapon, DepleteAmmo)
 	{
-		VMValue params[] = { (DObject*)this, AltFire, checkEnough, ammouse };
+		VMValue params[] = { (DObject*)this, altFire, checkEnough, ammouse };
```

Comment:
AltFire (with capital A) is an enum value
altFire (small a) is one of the parameters